ci-automation/Dockerfile

```diff
@@ -1,12 +1,6 @@
-FROM ubuntu:18.04
+FROM hrishikesh/libvirtd
 
-ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update \
-&& apt-get install -y vim rsync virtualbox wget openssh-server \
-&& wget -c https://releases.hashicorp.com/vagrant/2.2.0/vagrant_2.2.0_x86_64.deb \
-&& dpkg -i vagrant_2.2.0_x86_64.deb \
-&& vagrant box add centos/7 --provider virtualbox
-
-WORKDIR /root/ember-csi
-ENV VAGRANT_VAGRANTFILE ci-automation/Vagrantfile
-CMD ["vagrant", "up" ]
+RUN yum -y install rsync openssh-clients libvirt-client libxslt-devel libxml2-devel libvirt-devel libguestfs-tools-c ruby-devel gcc \
```

Comment:
For readability of the file, I suggest staying within the 80 lines limit

Reply:
+1